lib/actions/user.js

```diff
@@ -100,6 +101,11 @@ function setUser (user, fetchTrips) {
     if (fetchTrips) {
       dispatch(fetchMonitoredTrips())
     }
+
+    const { accessibilityRoutingByDefault } = user
+    if (accessibilityRoutingByDefault) {
+      dispatch(setQueryParam({wheelchair: accessibilityRoutingByDefault}))
```

Comment:
Yeah that's true that's a bit unclear. Have made it a bit more clear in e7578083ff6ddbdb2d25f1a7ca60964484260a07. I'm weary of doing an explicit true so that it's clear that the variable can be false too rather than just undefined.